src/main/java/de/adesso/kicker/configurations/TomcatConfig.java

```diff
@@ -0,0 +1,41 @@
+package de.adesso.kicker.configurations;
+
+import org.apache.catalina.Context;
+import org.apache.catalina.connector.Connector;
+import org.apache.tomcat.util.descriptor.web.SecurityCollection;
+import org.apache.tomcat.util.descriptor.web.SecurityConstraint;
+import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
+import org.springframework.boot.web.servlet.server.ServletWebServerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class TomcatConfig {
+
+    @Bean
+    public ServletWebServerFactory servletWebServerFactory() {
```

Comment:
I think I found [the source of inspiration](https://howtodoinjava.com/spring-boot/spring-boot-ssl-https-example/) for this example? ;D 

Assuming you really know what the code does, it's all looking good! 💯 

Bonus quest: How would you test this method? Don't forget to cover your code! ;D